klaviyo/profiles.py

```diff
@@ -94,3 +96,16 @@ def get_profile_metrics_timeline_by_id(self, profile_id, metric_id, since=None,
             self.HTTP_GET,
             params=filtered_params
         )
+
+    def get_profile_id_by_email(self, email):
+        """Gets the profile ID tied to a given email (if one exists).
+
+            Args:
+                email (str): Email to get profile ID for.
+            Returns:
+                (KlaviyoAPIResponse): Object with HTTP response code and data.
+        """
```

Comment:
unless there is still no docs for it sharing the whole url like what was done for other wrapper endpoints?

Reply:
Yeah I couldn't find any publically exposed documentation - if we have some I can totally add a link here!

Reply:
we don't have any haha